lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2425,8 +2461,34 @@ function( $pseudo_selector ) use ( $selector ) {
 		$declarations_duotone = array();
 		foreach ( $declarations as $index => $declaration ) {
 			if ( 'filter' === $declaration['name'] ) {
-				unset( $declarations[ $index ] );
+				$filter_preset = $this->gutenberg_get_preset_from_declaration( $declarations[ $index ] );
+				$filter_svg = gutenberg_get_duotone_filter_svg( $filter_preset );
+
+				add_action(
```

Comment:
One we have the SVG we then add it to the body of the page. This code also lives in block-supports/duotone.php so we should refactor to share the code.